src/cpp/rstudio-tests.in

```diff
@@ -8,6 +8,14 @@ checkUnitTestFailure() {
    fi
 }
 
+checkRUnitTestFailure() {
+    FAILURES=$(cat ${CMAKE_CURRENT_BINARY_DIR}/testthat-failures.log)
+    if [[ $FAILURES != 0 ]];
```

Comment:
Is that trailing semi-colon intentional? (I thought it was normally only used if `then` was on the same line)

Reply:
I put the then on the same line and then noticed the different style above and moved it to its own line. I'll remove the semicolon :)

Reply:
Further down in the file we keep the 'then' on the same line, so I updated both to be on the same line as well.